src/scr_util.c

```diff
@@ -389,3 +390,23 @@ spath* scr_get_prefix(const char* str)
 
   return prefix_path;
 }
+
+/* TODO: This needs to be moved somewhere else and probabbly renamed as well
+ */
+int scr_leader_rank(scr_filemap* map, const char *file)
```

Comment:
I think we could move this to scr_filemap.c.

Reply:
Done